testdata/customtemplates/header.tmpl

```diff
@@ -0,0 +1,10 @@
+{{define "header"}}
```

Comment:
What happens if you don't include these files? Does it crash? Do you need to duplicate all the files when you use custom templates?

Reply:
It currently crashes, but with implementing the comment after this that changes

Reply:
If it's no longer crashing, please remove a few the testdata template files that do not have any modifications from the originals. Ideally a user should only need to put a modified version of the one or two files they want to change in their custom-templates directory.